clients/drcachesim/tests/offline-tlb_simulator_v2p.templatex

```diff
@@ -21,9 +21,69 @@ Core #0 \(1 thread\(s\)\)
     Local miss rate:        *[0-9,.]*%
     Child hits:                *[0-9,\.]*
     Total miss rate:                  0[,\.]..%
-Core #1 \(0 thread\(s\)\)
-Core #2 \(0 thread\(s\)\)
-Core #3 \(0 thread\(s\)\)
+Core #1 \(2 thread\(s\)\)
+  L1I stats:
+    Hits:                      *[0-9,\.]*
+    Misses:                    *[0-9,\.]*
+    Compulsory misses:         *[0-9,\.]*
+    Invalidations:             *0
+    Miss rate:                        [0-3][,\.]..%
```

Comment:
(Hmm, other than these 0-3 lines, everything else is allowing anything: should we bother to list all the lines?  I thought we had collapsed most of these templates as it's a pain when we add new lines in the output.  OK to just leave it for now.)

Reply:
This is actually an offline trace, so I can put the exact output here, it shouldn't change from run to run.
Done.